portal-api/src/v1/services/utilisation-report/email-service.test.ts

```diff
@@ -0,0 +1,106 @@
+import { BankResponse } from 'src/v1/api-response-types';
+import {
+  sendUtilisationReportUploadConfirmationEmailToBankPaymentOfficerTeam,
+  sendUtilisationReportUploadNotificationEmailToUkefGefReportingTeam,
+} from './email-service';
+import sendEmail from '../../email';
+import { getBankById } from '../../api';
+import { EMAIL_TEMPLATE_IDS } from '../../../constants';
+import { aBank } from '../../../../test-helpers/test-data/banks';
+
+console.error = jest.fn();
+console.info = jest.fn();
+
+jest.mock('../../api');
+jest.mock('../../email');
+
+const originalProcessEnv = process.env;
+
+describe('emailService', () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+    process.env = { ...originalProcessEnv };
+  });
+
+  describe('sendUtilisationReportUploadNotificationEmailToUkefGefReportingTeam', () => {
+    it('should send utilisation report notification email to all emails belonging to UKEF GEF reporting team', async () => {
+      // Arrange
+      process.env.UKEF_GEF_REPORTING_EMAIL_RECIPIENT = '["email1@ukexportfinance.gov.uk", "email2@ukexportfinance.gov.uk"]';
+
+      // Act
+      await sendUtilisationReportUploadNotificationEmailToUkefGefReportingTeam('My Bank', 'June 2026');
+
+      // Assert
+      expect(sendEmail).toHaveBeenCalledTimes(2);
+      expect(sendEmail).toHaveBeenCalledWith(EMAIL_TEMPLATE_IDS.UTILISATION_REPORT_NOTIFICATION, 'email1@ukexportfinance.gov.uk', {
+        bankName: 'My Bank',
+        reportPeriod: 'June 2026',
+      });
+      expect(sendEmail).toHaveBeenCalledWith(EMAIL_TEMPLATE_IDS.UTILISATION_REPORT_NOTIFICATION, 'email2@ukexportfinance.gov.uk', {
+        bankName: 'My Bank',
+        reportPeriod: 'June 2026',
+      });
+    });
+
+    it('should throw an error if email recipients not provided in correct format', async () => {
+      // Arrange
+      process.env.UKEF_GEF_REPORTING_EMAIL_RECIPIENT = 'email1@ukexportfinance.gov.uk,email2@ukexportfinance.gov.uk';
+
+      // Act + Assert
+      await expect(
+        sendUtilisationReportUploadNotificationEmailToUkefGefReportingTeam('My Bank', 'June 2026'),
+      ).rejects.toThrow(Error);
```

Comment:
Do you think there is any benefit here to asserting that the error thrown is exactly what we expect rather than just checking that `Error` is thrown? I only ever include the `Error` constructor if I actually want to assert against the body of the error, like
```typescript
await expect(...).rejects.toThrow(new Error('my error message');
```
Otherwise, I tend to just use
```typescript
await expect(...).rejects.toThrow();
```
which I think does the same as the assertion we have here. Is there a benefit I'm missing with providing the plain `Error` object as an argument to `toThrow`?